metrics/mobile-app-metrics.md

```diff
@@ -0,0 +1,131 @@
+# Abstract
+
+This document contains the proposal for gathering metrics on system events for running Mobile Apps and making them available to developers.
+
+## Goals
+
+### Primary Goals
+
+- Provide mobile app developers a service that can receive overall runtime metrics (processor and memory data, authentication events, runtime versions, networking events, etc).
+
+- Provide developers an interface to query and visualize the gathered data in order to make informed decisions towards deployment, deprecation of backend services and other actions that depend on the state of an application's installed base.
+
+### Secondary Goals
+
+- Provide mobile app developers with an easy to use SDK to connect easily and securely to the metrics service
+
+### Non Goals
+
+- Identify individual devices or users in metrics events
+
+- Registering of user-defined metrics metrics
+
+## Terms
+
+For a list of terms used in this document, see [Glossary](./glossary.md).
+
+## Problem Description
+
+As mobile applications become more successful, system metrics data becomes invaluable for developers to observe the landscape of application installations and make sure further development and deployment of backend services will not affect users.
```

Comment:
This is good.
A small change I'd recommend
"As Mobile Apps are developed, improved and iterated on, different versions of those Apps can be used on many different devices. System metrics data ..."